core/sql/executor/ExExeUtilGetStats.cpp

```diff
@@ -2319,47 +2321,38 @@ short ExExeUtilGetRTSStatisticsTcb::work()
       {
         if (measStatsItems_ == NULL)
         {
-          maxMeasStatsItems_ = 35;
+          maxMeasStatsItems_ = 26;
```

Comment:
I think Arvind was correct. The number should be 25.  For me this caused a regression failure by outputting a couple spurious stats at the end. (an extra Accessed Rows stat where none should be)